packages/app-project/stores/User/UserPersonalization/YourStats/YourStats.spec.js

```diff
@@ -50,49 +62,29 @@ describe('Stores > YourStats', function () {
   })
 
   describe('Actions > fetchDailyCounts', function () {
-    let clock, statsNockScope
+    let clock
 
-    before(async function () {
+    before(function () {
       clock = sinon.useFakeTimers({ now: new Date(2019, 9, 1, 12), toFake: ['Date'] })
       const user = {
         id: '123',
         login: 'test.user'
       }
 
       rootStore.user.set(user)
-
-      const MOCK_DAILY_COUNTS = [
-        { count: 12, period: '2019-09-29' },
-        { count: 12, period: '2019-09-30' },
-        { count: 13, period: '2019-10-01' },
-        { count: 14, period: '2019-10-02' },
-        { count: 10, period: '2019-10-03' },
-        { count: 11, period: '2019-10-04' },
-        { count: 8, period: '2019-10-05' },
-        { count: 15, period: '2019-10-06' }
-      ]
-
-      statsNockScope = nock('https://eras-staging.zooniverse.org')
-      .get(`/classifications/users/${user.id}`)
-      .query({
-        'period': 'day',
-        'project_id': project.id
-      })
-      .reply(200, {
-        data: MOCK_DAILY_COUNTS
-      })
-
-      await when(() => rootStore.user.personalization.stats.loadingState === asyncStates.success)
     })
 
     after(function () {
       clock.restore()
     })
 
+    it('should request user statistics', function () {
+      expect(statsClient.fetchDailyStats).to.have.been.calledOnceWith({ projectId: '2', userId: '123' })
+    })
 
     describe('weekly classification stats', function () {
       it('should be created', function () {
-        expect(getSnapshot(rootStore.user.personalization.stats.thisWeek).length).to.equal(7)
+        expect(rootStore.user.personalization.stats.thisWeek.length).to.equal(7)
```

Comment:
I think the MST docs recommend testing with snapshots but I'm not sure if it makes a big difference here.